crapssim/bet.py

```diff
@@ -2,12 +2,67 @@
 import typing
 from abc import ABC, ABCMeta, abstractmethod
 from dataclasses import dataclass
-from typing import Protocol, TypedDict
+from typing import Hashable, Literal, Protocol, TypedDict, TypeAlias
 
 from crapssim.dice import Dice
 from crapssim.point import Point
 
+DicePair: TypeAlias = tuple[int, int]
+"""Pair of dice represented as (die_one, die_two)."""
+
+
+class SupportsFloat(Protocol):
+    """Protocol for objects that can be converted to ``float``."""
+
+    def __float__(self) -> float:
+        """Return a float representation."""
+
+
+def compute_commission(base_amount: float) -> float:
+    """Compute standard 5% commission for Buy/Lay style bets."""
+
+    return base_amount * 0.05
```

Comment:
Suggest removing this as a function and just copying the logic into `_compute_commission()`